QUEST_LV_0200.tsv

```diff
@@ -5226,53 +5226,53 @@ QUEST_LV_0200_20150323_005225	Hand over the joint signature seal
-QUEST_LV_0200_20150323_005256	Bokor Master told you that the spell spreads the poison that is on the the dagger. But, he told you that he doesn't know well about the dagger so told you to meet Ranger Master.
+QUEST_LV_0200_20150323_005253	Unfortunately, the Pyromancer Master doesn't know much about the spell. Talk to him again.
+QUEST_LV_0200_20150323_005254	Talk to the Bokor Master about the true nature of the spell.
+QUEST_LV_0200_20150323_005255	The Bokor Master knows something about the spells. Ask him about the true nature of the spells.
+QUEST_LV_0200_20150323_005256	The Bokor Master told you that the spell spreads the poison that is applied to the dagger. But, he told you that he is not familiar with the dagger so you should consult the Ranger Master.
 QUEST_LV_0200_20150323_005257	Talk to Gitise
 QUEST_LV_0200_20150323_005258	Use the antidote
-QUEST_LV_0200_20150323_005259	Cure the people who is suffering from the massive explosion using the antidote
-QUEST_LV_0200_20150323_005260	Talk to Pyromancer Master
-QUEST_LV_0200_20150323_005261	Talk to Creomancer Master
-QUEST_LV_0200_20150323_005262	Talk to the soldier, Denise
-QUEST_LV_0200_20150323_005263	You can see Denise who looks to be troubled. Talk to Denise.
+QUEST_LV_0200_20150323_005259	Use the antidote to cure the people who are suffering from the massive explosion
+QUEST_LV_0200_20150323_005260	Talk to the Pyromancer Master
+QUEST_LV_0200_20150323_005261	Talk to the Cryomancer Master
+QUEST_LV_0200_20150323_005262	Talk to Soldier Denise
+QUEST_LV_0200_20150323_005263	You notice that Denise is looking troubled. Talk to him.
 QUEST_LV_0200_20150323_005264	Defeat the monsters at the joint harvesting area
-QUEST_LV_0200_20150323_005265	Denise told you that the other soldiers went out to defeat Gorgon so it's hard to defeat the monsters that are in the farm. Defeat the monsters in the joint harvesting area.
+QUEST_LV_0200_20150323_005265	Denise told you that the other soldiers went out to defeat Gorgon so it's hard to fend off the monsters that are in the farm. Defeat the monsters in the joint harvesting area.
 QUEST_LV_0200_20150323_005266	You've defeated the monsters at the joint harvesting area. Go back to Denise and talk to him.
 QUEST_LV_0200_20150323_005267	Defeat the monsters at the farmland
-QUEST_LV_0200_20150323_005268	Denise still feels something's not right. Talk to Denise.
+QUEST_LV_0200_20150323_005268	Denise still feels that something is not right. Ask him about it.
 QUEST_LV_0200_20150323_005269	Talk to Alan
-QUEST_LV_0200_20150323_005270	Denise is worried about Alan who is protecting the entrance of the farm alone. Go to the entrance and support Alan.
-QUEST_LV_0200_20150323_005271	You've blocked against the monsters well. Go back to Denise and talk to him.
+QUEST_LV_0200_20150323_005270	Denise is worried about Alan who is protecting the entrance to the farm alone. Go to the entrance and support him.
```

Comment:
Hmm that doesn't quite sound right if you say out aloud (besides that you meant defend rather than defending).

Maybe to keep the context you could say: "Go to the entrance and defend the farm."
I think the best bet is just to replace *"support him."* with *"support Alan."*